composer.json

```diff
@@ -105,6 +105,11 @@
             "**/Tests/"
         ]
     },
+    "autoload-dev": {
+        "psr-4": {
+            "Symfony\\Component\\Console\\Tests\\": "src/Symfony/Component/Console/Tests"
+        }
+    },
```

Comment:
Classes are already correctly autoloaded, as they are just following the normal autoload configuration.